### ISSUE TITLE
wxPython always uses Python 2.7

### ISSUE THREAD

**Issue by edolstra** [NixOS/nixpkgs#9610]
Thus `python34Packages.wxPython` actually uses 2.7 rather than 3.4:
```
$ nix-store -qR $(nix-instantiate -A python34Packages.wxPython) | grep python-
/nix/store/xrig1svcrisl7fj2wikaw42ixdg93lsb-python-2.7.10.drv
```

(Noticed by accident, I don't actually need wxPython.)

**Comment by domenkozar** [comment]
see #9665

**Comment by vbgl** [comment]
Fixed in fba3a4e227d847bfe5238fcbb249f3ffc3aa1807.
